src/ServicePulse.Host/app/modules/monitoring/views/endpoint_details.html

```diff
@@ -35,10 +35,16 @@ <h1>{{endpointName}}</h1>
                              class="large-graph pull-left"></large-graph>
 
                 <div class="col-sm-12 no-side-padding graph-values">
-                    <div class="row queue-length-values">
-                        <span class="metric-digest-header" uib-tooltip="Queue length (# msgs): The estimated number of messages in an endpoint's queue.">
-                            Queue Length
-                        </span>
+                    <div class="row queue-length-values tooltip-trigger">
+                        <div class="metric-digest-header">
+                            Queue Length <i class="fa fa-flask fake-link"></i>
+                        </div>
+                        <div class="interactive-tooltip">
+                            <div class="tooltip-contents">
+                                <p>Queue length: The estimated number of messages in an endpoint's queue.</p>
+                                <p>NOTE: This is an experimental feature. <a href="https://docs.particular.net/nservicebus/operations/metrics/#metrics-captured-queue-length" target="_blank">Learn more</a> <i class="fa fa-external-link fake-link"></i></p>
```

Comment:
I think we can't do this. I remember talking to @Particular/docs-maintainers about creating links to our documentation and the decision was to not do it. The reason for that is increased complexity in maintaining doc urls. Unless we will get an explicit allowance for creating links to doco I am suggesting to remove this link or to link to google search results of a certain phrase.

Reply:
I would argue that the link is necessary here. My arguments are:
  * QL is experimental and want to point to exact section in a doco page - otherwise the link is useless IMHO,
  * This is to be removed when problems with Ql are solved.

Reply:
Thinking more critically about the "no links in source code" thing, I believe that's absolutely critical for NServiceBus because those versions stick around forever. I'm not sure if it's as critical in ServicePulse where customers are expected to upgrade more regularly, especially if it's a temporary thing.

Reply:
I'm not sure about that. Customers only seem to update when they log a support case and we ask them to update. If we can get away without direct links that would be better.

Reply:
@WilliamBZA I think that in this case a search query url is not good enough :/. What we need is specific [section](https://docs.particular.net/nservicebus/operations/metrics/#metrics-captured-queue-length). 
Otherwise it will be confusing to the users. I click to learn more why QL is experimental and I get to a search results page. That's confusing IMHO.

Reply:
A section that has to be maintained at the same url forever and never moved or deleted. I'd much rather we put a concise description in a tooltip if possible? I know it's difficult to fit all the required information in, but even just the note that's currently in the documentation page should be enough.

Reply:
I agree to disagree. I've changed the direct link to search query link.

Reply:
> I think we can't do this. I remember talking to @Particular/docs-maintainers about creating links to our documentation and the decision was to not do it. The reason for that is increased complexity in maintaining doc urls.

I think that the main concern was linking from inside the NSB code, as that indeed will be permanent (alternative would be to release a patch every time the link changes, we just won't do it). And if I recall that was mainly about linking to exceptions or more info about particular feature.

The general guideline is to avoid it, if possible. In many cases that can be achieved by crafting a meaningful comment or error message and giving user a "hint" what to look for. But I think in the context of a tool such as SP this is a common industry practice to link to help articles, etc. and I don't see a problem here.

Also note that maintainers in reviews were checking things like adding redirects to page headers when page name or location changed, so even if URL is different over time, that should not break the link. Assuming, we do that work properly (which I believe we do).

Personally, I think it's better to not give link than link to search results, it's not helpful for the user and is not what you'd expect after clicking a link. As for including link to a particular page from SP, I don't have objections. Just remember about redirects when updating the content ;)